api/gateway/middleware/rewrite/v1/rewrite.proto

```diff
@@ -13,5 +13,6 @@ message Rewrite {
     optional string path_rewrite = 1;
     HeadersPolicy request_headers_rewrite = 2;
     HeadersPolicy response_headers_rewrite = 3;
+    optional  int64 stripPrefix = 4;
```

Comment:
optional string stripPrefix = 4;

stripPrefix: /v1/prefix

path = strings.TrimPrefix(path, stripPrefix)

likes: https://pkg.go.dev/net/http#StripPrefix

Reply:
stripPrefix is a number stand for head parts of request's path

ref:
https://github.com/hshe/spring-cloud-gateway/blob/main/spring-cloud-gateway-server/src/main/java/org/springframework/cloud/gateway/filter/factory/StripPrefixGatewayFilterFactory.java#L69